SpinCore_pp/pulse_length_conv.py

```diff
@@ -1,79 +1,122 @@
-from pylab import array
 import numpy as np
-from pyspecdata import r_, nddata
+from numpy import r_
+import SpinCore_pp as spc
 
 
-def prog_plen(desired_actual):
+def prog_plen(desired_beta, amplitude, deblank_us):
     """
-    Takes the desired pulse length and tells the
-    user what pulse length should be programmed in order to get the actual desired
-    pulse length
+    Takes the desired β (μs√(W)), amplitude and deblank_us settings
+    to calculate what pulse length should be programmed in order to
+    get the desired β based on the amplitude and deblank setting.
+
     Parameters
     ==========
-    desired_actual: float
-                    the actual pulse length you wish the spincore to output,
-                    in us
+    desired_beta: float
+                    the desired β you wish the spincore to output,
+                    in μs*sqrt(W)
+    amplitude: float
+                amplitude setting of the output rf pulse
+    deblank_us: float
+                deblanking period before the actual output
+                rf pulse
```

Comment:
this was rolled back since we aren't making the config_dict an argument of echo.py it can't feed this function the config_dict so I just make these arguments flat out